workbench-core/authorization/README.md

```diff
@@ -5,7 +5,11 @@
 # Code Coverage
 | Statements                  | Branches                | Functions                 | Lines             |
 | --------------------------- | ----------------------- | ------------------------- | ----------------- |
+<<<<<<< HEAD
```

Comment:
This merge conflict was committed, but not resolved.

Reply:
odd, vscode didn't overwrite the changes

Reply:
fixed